src/components/Dropdowns/Dropdowns.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import './Dropdowns.scss';
+import { getSearchWith } from '../../helpers/getSearchWith';
+
+type Props = {
+  total: number,
+};
+
+export const Dropdowns: React.FC<Props> = ({ total }) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const [sortValue, setSortValue] = useState('Newest');
+  const [pageValue, setPageValue] = useState('4');
+  const [isOpen, setIsOpen] = useState(false);
+  const [isOpenPages, setIsOpenPages] = useState(false);
+
+  const toggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const togglePages = () => {
+    setIsOpenPages(!isOpenPages);
+  };
+
+  const onSelectSortOption = (option: string, value: string) => {
+    setSearchParams(
+      getSearchWith(searchParams, { sort: option }),
+    );
+
+    setSortValue(value);
+    toggle();
+  };
+
+  const onSelectPerPage = (option: string) => {
```

Comment:
create enum for this & use in all cases
```suggestion
  const onSelectPerPage = (option: SelectOptionEnum) => {
```